subsys/bluetooth/audio/codec.c

```diff
@@ -18,6 +20,74 @@
 
 LOG_MODULE_REGISTER(bt_audio_codec, CONFIG_BT_AUDIO_CODEC_LOG_LEVEL);
 
+int bt_audio_codec_cfg_freq_to_freq_hz(enum bt_audio_codec_config_freq freq)
+{
+	switch (freq) {
+	case BT_AUDIO_CODEC_CONFIG_LC3_FREQ_8KHZ:
+		return 8000U;
```

Comment:
Is it ok to return `unsigned int` to `int` ?

Reply:
Hmm, as long as it isn't larger than `INT_MAX` it shouldn't be an issue, right? But I can remove the `U` from the return values if you'd like

Reply:
yes, please remove... alternatively, please use const look up table array as `freq` is sequential series of values.

Reply:
Yeah I did consider a lookup table, but decided for a switch so that converting the other way around looked the same. Is there any memory optimizations from using a lookup table in this case? I guess there is a small performance gain going from O(n) to O(1)